source/common/http/http1/codec_impl.cc

```diff
@@ -439,6 +439,10 @@ Status RequestEncoderImpl::encodeHeaders(const RequestHeaderMap& headers, bool e
   }
   if (Utility::isUpgrade(headers)) {
     upgrade_request_ = true;
+    if (Runtime::runtimeFeatureEnabled(
+            "envoy.reloadable_features.http_skip_adding_content_length_to_upgrade")) {
+      disableChunkEncoding();
```

Comment:
note: this arguably isn't "safe" to flip from true to false live, as it'll corrupt any upgrade requests which have had headers read, but not sent.  I'm OK with this but I want to make sure we're doing it mindfully.

Reply:
Added comment